ui/page/page.go

```diff
@@ -25,4 +25,10 @@ type Page interface {
 const (
 	// StateWalletInfo is the map key for the WalletInfo state
 	StateWalletInfo = "walletinfo"
+
+	// StateWalletCreated is the map key for the WalletCreated state
+	StateWalletCreated = "walletCreated"
+
+	// StateError is the map key for error
+	StateError = "error"
```

Comment:
Should the window have one state for all window errors or multiple error states? (Not a suggestion, I'm actually asking coz I'm not sure which would be better)

Reply:
i strongly favor having one state for errors as it'd be cleaner as long as it's cleared when done with

Reply:
Okay